middle_end/flambda2/to_cmm/to_cmm_set_of_closures.ml

```diff
@@ -274,12 +274,28 @@ end = struct
                store code ID %a which is classified as \
                Full_and_partial_application (so the expected size is 3)"
               Function_slot.print function_slot size Code_id.print code_id;
+          let res, curry_code_pointer =
+            if only_known_arity
+            then
+              (* CR gbury: put the correct machtypes expected by the
+                 curry_code_pointer here *)
```

Comment:
we agreed that we do not need to do this in practice (particularly if we want to have a single function at one point to save code space), so instead we should put a comment here explaining that it's "okay" to put less parameters in the function declaration (compared to function calls), since the ocaml call convention is compatible with that (which is not the case for the C call convention for instance iirc).